README.md

```diff
@@ -54,7 +54,7 @@ Plugin versions and respective Velero/Kubevirt/CDI versions that are tested to b
 
 ## Install
 
-To install the plugin check current velero documentation https://velero.io/docs/v1.7/overview-plugins/.
+To install the plugin check current velero documentation https://velero.io/docs/v1.15/overview-plugins/.
```

Comment:
If this is the default main branch, let's use https://velero.io/docs/main/

Reply:
I agree with @weshayutin -- I think it's more appropriate here to reference main and reference a specific velero release on the release branch of the plugin, aligned to the newest Velero release that the plugin release is expected to support.

Reply:
ok, reverted to main.

Reply:
Okay makes sense, thanks!